apis/v1alpha2/shared_types.go

```diff
@@ -340,6 +340,12 @@ type AnnotationValue = v1beta1.AnnotationValue
 // +kubebuilder:validation:Pattern=`^Hostname|IPAddress|NamedAddress|[a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*\/[A-Za-z0-9\/\-._~%!$&'()*+,;=:]+$`
 type AddressType = v1beta1.AddressType
 
+// Duration is a string value representing a duration in time. The format is as specified
+// in GEP-2257, a strict subset of the syntax parsed by Golang time.ParseDuration.
+//
+// +kubebuilder:validation:Pattern=`^([0-9]{1,5}(h|m|s|ms)){1,4}$`
+type Duration v1beta1.Duration
```

Comment:
This should be a type alias: https://stackoverflow.com/questions/61247864/what-is-the-difference-between-type-alias-and-type-definition-in-go
```suggestion
type Duration = v1beta1.Duration
```

Reply:
addressed in d8f2b2bd1792c458eeb93fffee6c08f0392c05f6